notebooks/00_quick_start/lightgbm_tinycriteo.ipynb

```diff
@@ -0,0 +1,817 @@
+{
```

Comment:
TREE_LEARINING_RATE

Reply:
How about maximum depth of trees? It is an important parameter, you could mention the default value.

Reply:
Could we use one-hot encoding for the categorical features or would it be too slow?